lib/widgets/compass/compass_painter.dart

```diff
@@ -0,0 +1,197 @@
+import 'dart:math';
+
+import 'package:chirp_nets/models/user.dart';
+import 'package:chirp_nets/utils/theme.dart';
+import 'package:chirp_nets/utils/utils.dart';
+import 'package:flutter/material.dart';
+
+enum directions {
+  north,
+  south,
+  east,
+  west,
+}
+
+class CompassPainter extends CustomPainter {
```

Comment:
Are these methods scalable for all phone sizes?

Reply:
It'll need work to get it scalable as you've mentioned a lot of sizes are hardset

Reply:
perfect. Moving forward it certainly makes sense to ensure all our screens are properly scalable. Perhaps this can be fixed on the compass screen before merging to master.